docs/how-to-guides/deploy.md

```diff
@@ -6,22 +6,78 @@ id: deploy
 
 ## Overview
 
-This guide shows you how to deploy any realm or package to the Gno chain. Deployment is be done by utilizing `gnokey`'s `maketx addpkg` API.
+This guide shows you how to deploy any realm or package to the Gno chain. It will
+show you how to:
+- Deploy Gno code in your browser via the Playground, 
+- Deploy Gno code from your local machine using `gnokey`'s `maketx addpkg` API.
+
+
+## Deployment via the Playground
+
+Deployment via the Playground is recommended for smaller realms and packages.
+For larger projects, it is recommended to write, test, and deploy your code from
+a local environment.
+For this, check out [**Deployment from a local environment**](#deployment-from-a-local-environment).
+
+### Prerequisites
+
+- **Internet connection**
+- **A keypair in a Gno.land wallet, such as [Adena](https://adena.app)**
+
+### Using Gno Playground
+
+You can write, test, and deploy packages and realms using Gno Playground.
+To start using the Playground, you can check out XYZ.
+
+For this example, we will be using the **Counter** realm. You can find the code
+on [this Playground link](https://play.gno.land/p/iUWTha99D1J).
+
+Once you have confirmed that the code is without bugs, you can click on "**Deploy**".
```

Comment:
```suggestion
Review the code and when you're ready, click on "**Deploy**".
```

Reply:
Because the example code is meant to be instructive (and presumably without bugs)